basic_pitch/__init__.py

```diff
@@ -15,13 +15,81 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import enum
+import logging
 import pathlib
 
-__author__ = "Spotify"
-__version__ = "0.3.0"
-__email__ = "basic-pitch@spotify.com"
-__demowebsite__ = "https://basicpitch.io"
-__description__ = "Basic Pitch, a lightweight yet powerful audio-to-MIDI converter with pitch bend detection."
-__url__ = "https://github.com/spotify/basic-pitch"
 
```

Comment:
doesn't seem like we needed to get rid of this in adding the new stuff. What's the reason for the removal?

Reply:
Not really needed. It's all in pyproject.toml